.gitignore

```diff
@@ -1,3 +1,4 @@
 packages/**/dist
 node_modules
-.idea/discord.xml
+.idea
```

Comment:
Removed vcs.xml and workspace.xml because they're also not important
- vcs.xml: auto-generated upon launch
- workspace.xml: store things like last opened file

Reply:
`vcs.xml` should not be removed. We don't use it for any specific settings at the moment, but it can contain important configuration options.

`workspace.xml` is already ignored by `.idea/.gitignore`, which Jetbrains editors add by default.

For an exhaustive list of everything that should be ignored by ALL Jetbrains editors (i.e. we do not need anywhere near this many ignored files/folders), see this list:
https://github.com/github/gitignore/blob/main/Global/JetBrains.gitignore

Reply:
As discussed, turns out this was my issue. All idea config files are corrected.